File: src/main/kotlin/github/joeyslalom/howto/_01_bean/java/Bean.java
package github.joeyslalom.howto._01_bean.java;

public class Bean {
    private final String name;
    private final int age;

    public Bean(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }
}

File: src/main/kotlin/github/joeyslalom/howto/_01_bean/Bean.kt
package github.joeyslalom.howto._01_bean.kt

// fix
class Bean {
    private var name: String
    private var age: Int

    constructor(n: String, a: Int) {
        name = n
        age = a
    }

    fun getName(): String {
        return name
    }

    fun getAge(): Int {
        return age
    }
}